python-sdk/nuscenes/eval/lidarseg/README.md

```diff
@@ -0,0 +1,209 @@
+|   16                          |   vegetation                  |   static.vegetation                       |
+
+
+## Evaluation metrics
+Below we define the metrics for the nuScenes lidar segmentation task.
+Our final score is a weighted sum of mean intersection-over-union (IOU) and boundary IOU.
+
+### Preprocessing
+N.A.
+
+### Mean IOU (mIOU)
+We use the well-known IOU metric, which is defined as TP / (TP + FP + FN). 
+The IOU score is calculated separately for each class, and then the mean is computed across classes.
+
+### Boundary IOU (bIOU)
+To better evaluate the performance of the predicted segmentation, we introduce an additional metric to measure the accuracy for boundary delineation.
+Here, we project the lidar point cloud into the range view and determine which "pixels" in the range view image are boundary "pixels".
+We then define boundary IOU as TP / (TP + FP + FN) but taken only on the boundary "pixels".
+This is inspired by the work done by [Milioto et al.](http://www.ipb.uni-bonn.de/wp-content/papercite-data/pdf/milioto2019iros.pdf)
+Note that the boundary IOU is class-agnostic, i.e. it is **not** evaluated per class.
+
+### nuScenes-lidarseg score
+* **nuScenes-lidarseg score (NLS)**:
+We consolidate the above metrics by taking the average of mIOU and bIOU (i.e. equal weighting is given to both).
+
+### Configuration
+The default evaluation metrics configurations can be found in `nuscenes/eval/detection/configs/detection_cvpr_2019.json`. ## TODO
+
+## Leaderboard
+nuScenes will maintain a single leaderboard for the lidar segmentation task.
+For each submission the leaderboard will list method aspects and evaluation metrics.
+Method aspects include input modalities (lidar, radar, vision), use of map data and use of external data.
+To enable a fair comparison between methods, the user will be able to filter the methods by method aspects.
+
+Methods will be compared within these tracks and the winners will be decided for each track separately.
+Furthermore, there will also be an award for novel ideas, as well as the best student submission.
+
+**Lidar track**: 
+* Only lidar input allowed.
+* External data or map data <u>not allowed</u>.
+* May use pre-training.
+  
+**Open track**: 
+* Any sensor input allowed.
+* External data and map data allowed.  
+* May use pre-training.
+
+**Details**:
+* *Sensor input:*
+For the lidar track we restrict the type of sensor input that may be used.
+Note that this restriction applies only at test time.
+At training time any sensor input may be used.
+
+* *Map data:*
+By `map data` we mean using the *semantic* map provided in nuScenes. 
+
+* *Meta data:*
+Other meta data included in the dataset may be used without restrictions.
+E.g. calibration parameters, ego poses, `location`, `timestamp`, `num_lidar_pts`, `num_radar_pts`, `translation`, `rotation` and `size`.
+Note that `instance`, `sample_annotation` and `scene` description are not provided for the test set.
```

Comment:
Mention that nuScenes box annotations may be used during training.
Adapt for nuScenes-lidarseg. Add the bin files, besides `sample_annotation`.